📝 Update error tracking docs for new flutter-symbols command (#247)
DataDog/dd-sdk-flutter: packages/datadog_flutter_plugin/doc/rum/error_tracking.md

Enable Crash Reporting and Error Tracking to get comprehensive crash reports and error trends with Real User Monitoring. 

Your crash reports appear in [**Error Tracking**][1].

## Setup

If you have not set up the Datadog Flutter SDK for RUM yet, follow the [in-app setup instructions][2] or see the [Flutter setup documentation][3].

> ### Add Native Crash Reporting

Reviewer: ```suggestion
### Add native crash reporting
```

DatadogSdk.instance.initialize(configuration);
```

> If your application suffers a fatal crash, once your application restarts, the Datadog SDK uploads a crash report to Datadog. For non-fatal errors, the Datadog SDK will upload these errors with other RUM data. 

Reviewer: ```suggestion
If your application suffers a fatal crash, once your application restarts, the Datadog Flutter SDK uploads a crash report to Datadog. For non-fatal errors, the Datadog Flutter SDK uploads these errors with other RUM data. 
```

> ##  Upload Symbol Files to Datadog

Reviewer: ```suggestion
## Upload symbol files to Datadog
```

Native iOS crash reports are collected in a raw format and mostly contain memory addresses. To map these addresses into legible symbol information, Datadog requires that you upload .dSYM files, which are generated in your application's build process.

> All crash reports that are built with the `--split-debug-info` option set and / or with the `--obfuscate` option set will need to upload their Android Proguard mapping file, and the Dary symbol files generated by the Flutter build process.

Reviewer: ```suggestion
For all crash reports that are built with the `--split-debug-info` option set and/or with the `--obfuscate` option set, you need to upload their Android Proguard mapping file and Dart symbol files generated by the Flutter build process.
```

The [@datadog/datadog-ci][4] command line tool supports uploading all of the necessary files (dSYMs, Android Proguard Mapping, and Dart Symbol Files) in one command.

> First, install the `datadog-ci` tool from the instructions above, and create a `datadog-ci.json` file at the root of your project, containing your API key and (optionally) your Datadog site:

Reviewer: ```suggestion
First, install the `datadog-ci` tool from the instructions above and create a `datadog-ci.json` file at the root of your project, containing your API key and (optionally) your Datadog site:
```

> ```sh

Reviewer 2: ```suggestion
```json
```

{
  "apiKey": "<YOUR_DATADOG_API_KEY>",
  "datadogSite": "datadoghq.eu"  // Optional if you are using datadoghq.com
}
```

Because this file contains your API key, it should not be checked in to version control.

Alternately, you can set the `DATADOG_API_KEY` and `DATADOG_SITE` environment variables. 

Then, you can use the following command to upload all the necessary files for symbolication and deobfuscation of your crash reports:
```sh

> datadog-ci flutter-symbols upload --service-name <your_service_name> --dart-symbols-location <location_of_dart_symbols> --android-mapping --ios-dsyms

Reviewer 2: why dSyms are mentioned in both `datadog-ci flutter-symbols` call and `npx @datadog/datadog-ci dsyms upload` call?

Author: Looks like there's some lines tat the end of the readme that shouldn't be there. There shouldn't be any mention of `dsyms upload`

```

> For a full list of options, refer to the `datadog-ci` [flutter-symbols documentation][6]

Reviewer: ```suggestion
For a full list of options, see the `datadog-ci` [Flutter Symbols documentation][6].
```
